examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>I suggest adding a hyperlink to an external source of Black-Scholes model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done